### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by secosan**

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -91,19 +91,19 @@ module.exports = {
...
       resolve: 'gatsby-source-graphql',
       options: {
-        typeName: process.env.GATSBY_HASURA_GRAPHQL_TYPE_NAME,
-        fieldName: process.env.GATSBY_HASURA_GRAPHQL_FIELD_NAME,
-        url: process.env.GATSBY_HASURA_GRAPHQL_URL,
-        refetchInterval: 60,
+        typeName: 'MyQuery',
+        fieldName: 'hasura',
+        url: 'https://hasura.serviciisociale.ro/v1/graphql',
+        refetchInterval: 10,
```
Comment (cezarneaga) [inline gatsby-config.js]: pls revert back these

File: src/components/CircularProgressBar/index.tsx [hunk 2]
```diff
@@ -1,84 +1,85 @@
 import React from 'react';
+type Props = {
+  sqSize: number;
+  strokeWidth: number;
+  public: number;
+  private: number;
+  class: string;
```
Comment (cezarneaga) [inline src/components/CircularProgressBar/index.tsx]: change this to something else than `class`

File: src/components/ProgressBar/index.tsx [hunk 3]
```diff
@@ -1,64 +1,73 @@
 import React from 'react';
 import siluette from '../../assets/images/siluette.svg';
+type Props = {
+  sqSize: number;
+  shelters: number;
+  skids: number;
+  strokeWidth: number;
+  class: string;
```
Comment (cezarneaga) [inline src/components/ProgressBar/index.tsx]: same as above

File: src/components/ProgressBar/index.tsx [hunk 4]
```diff
@@ -1,64 +1,73 @@
...
+        <svg width={props.sqSize} height={props.sqSize} viewBox={viewBox} className="xl:m-auto">
           <line
             className="circle-background"
             y1="260"
             y2="260"
             x1="0"
-            x2={this.props.copii}
-            transform={`rotate(-90 ${this.props.sqSize / 2} ${this.props.sqSize / 2})`}
-            strokeWidth={`${this.props.strokeWidth}px`}
+            x2={props.skids}
```
Comment (cezarneaga) [inline src/components/ProgressBar/index.tsx]: skids? confused about naming here

File: src/components/Separator/index.tsx [hunk 5]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+type Props = {
+  color: string;
+  class: string;
```
Comment (cezarneaga) [inline src/components/Separator/index.tsx]: same as above

File: src/components/pages/HomePage/Sections/Hero.tsx [hunk 6]
```diff
@@ -1,13 +1,14 @@
 import Data from '../../../../data/global.json';
-import hero from '../../../../assets/images/hp_hero.png';
+import herod from '../../../../assets/images/HSS_people_clean_2000.png';
 import React from 'react';
+import Separator from '../../../Separator';
 
 export default class Content extends React.Component {
```
Comment (cezarneaga) [inline src/components/pages/HomePage/Sections/Hero.tsx]: function component refactor

**Review by cezarneaga — COMMENTED** [review]
am verificat rapid niste chestii de rezolvat

File: src/components/CircularProgressBar/index.tsx (hunk 2 above)
Comment (secosan) [reply src/components/CircularProgressBar/index.tsx]: done, will push in seconds

File: src/components/ProgressBar/index.tsx (hunk 3 above)
Comment (secosan) [reply src/components/ProgressBar/index.tsx]: done, will push in seconds

File: src/components/ProgressBar/index.tsx (hunk 4 above)
Comment (secosan) [reply src/components/ProgressBar/index.tsx]: shelterkids era, ai vreo alta propunere, am vrut sa ramana si evident ce e ce

Comment (cezarneaga) [reply src/components/ProgressBar/index.tsx]: ProgressBar ar trebui sa aiba props abstracte. nu putem folosi ceva generic ca sa putem folosi componenta si cua lte date?

File: src/components/pages/HomePage/Sections/Hero.tsx (hunk 6 above)
Comment (secosan) [reply src/components/pages/HomePage/Sections/Hero.tsx]: done, will push in seconds

File: src/components/Separator/index.tsx (hunk 5 above)
Comment (secosan) [reply src/components/Separator/index.tsx]: done, will push in seconds

File: src/components/ProgressBar/index.tsx (hunk 4 above)
Comment (secosan) [reply src/components/ProgressBar/index.tsx]: got it, on it

File: src/components/BlogPost/index.tsx [hunk 7]
```diff
@@ -1,27 +1,34 @@
...
+type Props = {
+  classBlogPost: string;
+  title: string;
+  date: string;
+  content: string;
+};
+
+function BlogPost(props: Props) {
+  return (
+    <div className={`${props.class} relative blog-post md:w-1/2 md:mx-2`}>
```
Comment (cezarneaga) [inline src/components/BlogPost/index.tsx]: ai uitat asta

File: src/components/ProgressBar/index.tsx [hunk 8]
```diff
@@ -2,8 +2,8 @@ import React from 'react';
 import siluette from '../../assets/images/siluette.svg';
 type Props = {
   sqSize: number;
-  shelters: number;
-  skids: number;
+  firstbar: number;
```
Comment (cezarneaga) [inline src/components/ProgressBar/index.tsx]: camelCase pls :) `firstBar`

**Review by cezarneaga — APPROVED** [review]

**Pull request merged**